src/Proxy.php

```
@@ -50,6 +50,8 @@ final class Proxy implements \Stringable, ProxyBase
     public function __construct(
         /** @param TProxiedObject $object */
         private object $object,
+
+        private bool $shouldPersist = true,
```

I think we should directly disable autorefrieshing when `shouldPersist` is false

---

maybe we should also add a condition in methods `_delete()`, `_refresh()`. And maybe throw an exception in `_disableAutoRefresh()`?